doc/build.rst

```diff
@@ -107,13 +105,20 @@ The minimal building requirement is
 Building the Targets
 ==================
 
-1. Clone the repository recursively:
+1. **Clone the repository recursively**:
 
    .. code-block:: bash
 
       git clone --recursive https://github.com/mc2-project/mc2-xgboost.git
 
-2. Configure the enclave parameters listed in ``CMakeLists.txt``; these parameters are used by the Open Enclave SDK to configure the enclave build.
+2. **Configure the build parameters** listed in ``CMakeLists.txt``. 
```

Comment:
It's a bit strange that only a part of the step is bolded. Can we make the entire line bold?